data/language/en-GB.txt

```diff
@@ -3698,6 +3698,8 @@ STR_6623    :Type ‘help’ for a list of available commands. Type ‘hide’ t
 STR_6624    :Tile Inspector: Sort elements
 STR_6625    :Invalid colour
 STR_6626    :Animation is backwards
+STR_6627    :Touch mode
```

Comment:
I think `Touch mode` implies too much, like support for dragging or pinching gestures. Could it be named something like `enlarged UI` or `2x UI`?

Reply:
I do want to add features like that in the future, though. E.g. adding buttons to raise/lower ghost scenery (stuff you currently use SHIFT for), or having a click event place a ghost, rather than placing for real.

Reply:
I have ultimately chosen to rename it to Enlarged UI after all.